src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconExclamationMarkCircle24, IconExclamationMarkTriangle24, IconTrashBin24 } from '@foundation/Icon';
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import { Button, ButtonGroup, ButtonSize, ButtonStyle } from '..';
+import { Toast as ToastComponent } from './Toast';
+import { ToastAnimationDirection, ToastProps, ToastStyle } from './types';
+
+export default {
+    title: 'Components/Toast',
+    component: ToastComponent,
+    tags: ['autodocs'],
+} as Meta;
+
+const WithLeftBottomContainer: StoryFn<ToastProps> = (args) => (
+    <div className="tw-fixed tw-bottom-0 tw-left-0 tw-m-3">
+        <ToastComponent {...args} />
+    </div>
+);
+
+const WithRightTopContainer: StoryFn<ToastProps> = (args) => (
+    <div className="tw-fixed tw-top-0 tw-right-0 tw-m-3">
+        <ToastComponent {...args} />
+    </div>
+);
+
+const WithRightBottomContainer: StoryFn<ToastProps> = (args) => (
+    <div className="tw-fixed tw-bottom-0 tw-right-0 tw-m-3">
+        <ToastComponent {...args} />
+    </div>
+);
+
+export const InformativeToastAtTheBottom = WithLeftBottomContainer.bind({});
+InformativeToastAtTheBottom.args = {
+    isOpen: true,
+    style: ToastStyle.Loud,
```

Comment:
I would add argTypes to the story with the possibility to select between all style options, and animationDirection as well.

![Screenshot 2023-03-02 at 15 32 14](https://user-images.githubusercontent.com/35405438/222457630-f13e8ec1-171a-4720-a247-20c32851f5ba.png)

Because now it's just a string. Would be better to display all options available.